gnovm/stdlibs/std/package.go

```diff
@@ -0,0 +1,19 @@
+package std
+
+import (
+	"github.com/gnolang/gno/tm2/pkg/amino"
+	abci "github.com/gnolang/gno/tm2/pkg/bft/abci/types"
+)
+
+var Package = amino.RegisterPackage(amino.NewPackage(
```

Comment:
Hello Nice PR :)
I'm was trying to understand your PR and reading some documentation about Amino 

Shouldn't this file be called amino.go by convention ?
https://github.com/gnolang/gno/blob/master/tm2/pkg/amino/README.md?plain=1#L61-L62

Reply:
Sorry I saw everywhere is the same

Reply:
Since other files are also using the `package` as a file name, so I used this instead of `amino` to maintain consistency. If we change the this file name, we need to change other files too. I think this work is outside the scope of the PR.
I believe we need to discuss separately whether to update the convention or change all the package names to follows the document. Anyway thanks for the comment